dockerfiles/share_task_html_report.dockerfile

```diff
@@ -0,0 +1,26 @@
+############################################################
+# Dockerfile for BROAD GRO share-seq-pipeline
+# Based on Ubuntu 18.04.3
+############################################################
+
+# Set the base image to Ubuntu 18.04.3
+#FROM ubuntu:focal
+FROM ubuntu@sha256:d1d454df0f579c6be4d8161d227462d69e163a8ff9d20a847533989cf0c94d90
+
+MAINTAINER Neva Durand
+
+# To prevent time zone prompt
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install softwares from apt repo
+RUN apt-get update && apt-get install -y \ 
+    python3 \
+    && rm -rf /var/lib/apt/lists/*
+
+# Make directory for all softwares
+RUN mkdir /software
+WORKDIR /software
+ENV PATH="/software:${PATH}"
+
+# Copy the external scripts inside
+COPY src/python /software
```

Comment:
Could you please just add the specific file you are using? I started to change this in the other dockers as well.